src/components/icons/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FunctionComponent } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+interface IconProps {
+  width?: string
+  height?: string
+  color?: string
+}
+
+const SvgWrap = styled.svg`
+  fill: ${({ theme }) => theme.text4};
+`
+
+export const RollBackIcon: FunctionComponent<IconProps> = ({ color, width = '16' }): React.ReactElement => {
+  return (
+    <SvgWrap
+      className="icon"
+      width={width}
+      height={width}
+      viewBox="0 0 16 16"
+      fill={color}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path fillRule="evenodd" d="M8 3a5 5 0 1 1-4.546 2.914.5.5 0 0 0-.908-.417A6 6 0 1 0 8 2v1z" />
+      <path d="M8 4.466V.534a.25.25 0 0 0-.41-.192L5.23 2.308a.25.25 0 0 0 0 .384l2.36 1.966A.25.25 0 0 0 8 4.466z" />
+    </SvgWrap>
+  )
+}
+
+RollBackIcon.propTypes = {
+  color: PropTypes.string,
+  width: PropTypes.string,
+}
+
+export const QuestionIcon: FunctionComponent<IconProps> = ({ color, width = '16' }): React.ReactElement => {
```

Comment:
Same point as the copy icon. There is already another question icon already used in the project

Reply:
agree, replaced with existing component